lib/ApiRequestor.php

```diff
@@ -312,11 +430,19 @@ private function _interpretResponse($rbody, $rcode, $rheaders)
         return $resp;
     }
 
+    /**
+     * @static
+     *
+     * @param HttpClient\CurlClient $client
```

Comment:
I missed this before, but could you also set this one as `HttpClient\ClientInterface`?